opal/include/opal/sys/atomic_stdc.h

```diff
@@ -48,12 +48,20 @@ static inline void opal_atomic_wmb(void)
     atomic_thread_fence(memory_order_release);
 }
 
+/* work around a bug in older gcc versions (observed in gcc 6.x)
+ * where acquire seems to get treated as a no-op instead of being
+ * equivalent to __asm__ __volatile__("": : :"memory") on x86_64
+ * [JS] the issue has still been observed in GCC 7.3.0 but appears
+ *      to be fixed from 8.1.0 onwards
+ */
+#   if  defined(PLATFORM_ARCH_X86_64) &&                        \
+        PLATFORM_COMPILER_GNU && __GNUC__ < 8
```

Comment:
Is the issue fixed in GCC 8.0 (the code) or GCC 8.1 (the comment)?  We should be careful there.  Do we need to check `__GNUC_MINOR__` as well?

Reply:
There was no 8.0 release. The first release in the 8 series was 8.1: https://gcc.gnu.org/releases.html